packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -31,65 +31,29 @@
-              v-if="slots['video']"
-              class="border-transparent rounded mb-6 bg-gray-50/50 border-4px text-center max-w-80vw w-688px overflow-hidden"
-            >
-              <div
-                class="bg-white rounded-t border-4px-gray-500 w-full p-24px"
-                data-cy="video-container"
-              >
-                <slot name="video" />
-              </div>
-            </div>
-
             <div class="mb-[16px]">
               <ExternalLink
-                href="https://on.cypress.io/changelog?utm_source=Binary%3A+App&utm_medium=splash-page&utm_campaign=v13#13-0-0"
+                href="https://on.cypress.io/changelog?utm_source=Binary%3A+App&utm_medium=splash-page&utm_campaign=v14#14-0-0"
                 class="font-bold text-indigo-500"
               >
-                13.0.0
+                14.0.0
               </ExternalLink>
               <span class="font-light pl-[10px] text-gray-500 text-[14px]">
-                Released {{ versionReleaseDates['13'] }}
+                Released {{ versionReleaseDates['14'] }}
               </span>
             </div>
             <div class="children:mb-[16px]">
               <p>
-                For a complete list of updates please review our
-                <ExternalLink href="https://on.cypress.io/changelog?utm_source=Binary%3A+App&utm_medium=splash-page&utm_campaign=v13#13-0-0">
-                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  changelog</ExternalLink>. To take a deep-dive into this release, read our <ExternalLink href="https://on.cypress.io/cypress-13-release?utm_source=Binary%3A+App&utm_medium=splash-page&utm_campaign=v13#13-0-0">
-                  <!--eslint-disable-next-line vue/multiline-html-element-content-newline-->
-                  blog post</ExternalLink>.
+                We’ve improved performance of component testing and added support for new framework and dev server versions.
               </p>
-
-              <h2 class="font-bold text-[18px] text-jade-1000">
-                Test Replay
-              </h2>
-
               <p>
-                Test Replay brings the debugging experience you know and love from the Cypress app directly into your recorded tests in Cypress Cloud. Previously, trying to debug failures and flake in CI was painful and time consuming with only videos & screenshots. Test Replay provides a way to inspect the DOM, network events, and console logs of your application from your tests exactly as they ran in CI.
+                This release also includes breaking changes to <InlineCodeFragment>cy.origin</InlineCodeFragment> that are necessary to handle
+                Chrome’s deprecation of <InlineCodeFragment>document.domain</InlineCodeFragment>, this should fix issues for some users in recent Chrome versions. Support for older versions of Node.js, Linux distributions, browsers and component testing frameworks and dev servers is also removed.
```

Comment:
```suggestion
                Chrome’s deprecation of <InlineCodeFragment>document.domain</InlineCodeFragment>, which should fix issues for some users in recent Chrome versions. Support for older versions of Node.js, Linux distributions, browsers and component testing frameworks and dev servers is also removed.
```

I think otherwise you would need a new sentence starting with ", this should fix ,,,"